CHANGELOG.md

```diff
@@ -6,20 +6,23 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to
 [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## unreleased
 
-+ fix `portlet-icon` directive to support `iconUrl`
+## 10.2.2 - 2020-07-22
```

Comment:
This looks correct, that the next release is 10.2.2, because [the most recent release was 10.2.1 on 2020-04-08](https://search.maven.org/artifact/org.apereo.uportal/uportal-home-parent).